### PR TITLE
garbage collector

### LINKED ISSUE #121: var/checkouts garbage collector
there is some collected on if the process is aborted.

On the test system we had ~1 GB

### PR DESCRIPTION
**Pull request by DavidBadura**
fixes #121

### CODE REVIEW

**Comment by tolry** [comment]
Code looks good, but I would have imagined this being a separate command. Running this after processing commits is a good idea, but I think it would be better to have an option for this, or at least an option to skip the garbage collector. Can be enabled by default.

And having additionally a separate command would still be nice imo :smile:

**Comment by DavidBadura** [comment]
you're right. i will do it

**Comment by DavidBadura** [comment]
@simpspector/owners done

File: src/SimpleThings/AppBundle/Command/JobRunCommand.php [hunk 1]
```diff
@@ -45,5 +54,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $commitHandler->handle($commit);
         }
 
+        if (!$input->getOption('no-garbage-collector')) {
+            $output->writeln('run garbage collector');
```
Comment (tolry) [inline src/SimpleThings/AppBundle/Command/JobRunCommand.php]: this should probably be present progressive "running garbage collector" :smile:

**Comment by tolry** [comment]
:+1: from me

But I would prefer naming the option "skip-garbage-collector" or "skip-garbage-collection". The if-statement right now reads a bit complicated due to the double negative "if not no-garbage-collector". I think "if not skip-garbage-collector" would be an improvement.

**Comment by DavidBadura** [comment]
I have kept to other projects like symfony or composer.

see 

https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php#L36

https://github.com/composer/composer/blob/master/src/Composer/Command/UpdateCommand.php#L29

somehow it seems to be a standard.

**Comment by tolry** [comment]
"no-#module-or-action-name#" is very technical, I think and definitely makes sense, if you have the inverse option, so you know how to disable an option explicitely when one only knows the positive option. In this case I think "skip" would fit better, but as I said, this is just a preference, :+1: still stands. 

We just need the second +1 - @simpspector/owners?

**Comment by larsborn** [comment]
+1

**Pull request merged**